src/Propel/Generator/Schema/Dumper/XmlDumper.php

```diff
@@ -503,8 +503,10 @@ private function appendIndexNode(Index $index, \DOMNode $parentNode)
     /**
      * Appends the generated <unique> XML node to its parent node.
      *
-     * @param Unique   $unique     The Unique model instance
+     * @param Unique   $index
```

Comment:
But as I said: Out of scope now of course.
The changes are already getting a bit too much.